ntbs-service/Pages/Notifications/OverviewPartials/_TestResultsOverviewPartial.cshtml

```diff
@@ -13,15 +14,75 @@
-            <div class="cell-min-height"> @Model.HasTestCarriedOut.FormatYesNo() </div>
+            <div class="notification-details-label"> @Html.DisplayNameFor(m => m.Notification.TestData.HasTestCarriedOut) </div>
+            <div class="cell-min-height"> @testData.HasTestCarriedOut.FormatYesNo() </div>
         </nhs-grid-column>
     </nhs-grid-row>
     
-    @if (Model.HasTestCarriedOut == true)
+    @if (testData.HasTestCarriedOut == true)
     {
         ViewData["RenderReducedSizeManualResultsTable"] = true;
         
-        <partial name="Shared/_ManualTestResultTable" for="ManualTestResults" />
+        <partial name="Shared/_ManualTestResultTable" for="Notification.TestData.ManualTestResults" />
+    }
+
+    @if (Model.CultureAndResistance != null)
+    {
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Culture Positive </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.CulturePositive </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Species </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.Species </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Drug Resistance </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.DrugResistanceProfile </dd>
+            </nhs-grid-column>            
+        </nhs-grid-row>
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Earliest Date </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.FormattedEarliestSpecimenDate </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Isoniazid </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.Isoniazid </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Rifampicin </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.Rifampicin </dd>
+            </nhs-grid-column>
+        </nhs-grid-row>
+        <nhs-grid-row>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Pyrazinamide </dt>
+                <dd class="cell-min-height"> @Model.CultureAndResistance.Pyrazinamide </dd>
+            </nhs-grid-column>
+            <nhs-grid-column grid-column-width="OneThird">
+                <dt class="notification-details-label"> Ethambutol </dt>
```

Comment:
Is this stuff the same as what's in the `_LabResults.cshtml`? If so, can they share a partial?

Reply:
a bit different, this is 3 in a row which is for Overview. The other one is 4 in a row

Reply:
ah right.. if we were using a single row (as discussed not quite feasible due to the row behaviour on differently-highted items) ,then i would have suggested using same partial and selecting between "1/3" and "1/4" classes as needed, but alas.

We should definitely use property display names in that case though!